### PR TITLE
Update ggraph examples

### PR DESCRIPTION
**Pull request by thomasp85**
This adds a bunch of examples showing the different possibilities with ggraph

### CODE REVIEW

**Comment by Emaasit** [comment]
@thomasp85 Thanks for this PR. I will review it tonight after class. I also wanted to invite you to co-admin this repo and the website with me. Let me know your thoughts. Thanks

**Comment by thomasp85** [comment]
I'll be happy to do that, but don't expect too much of me until after April, where I hand in my PhD... :-)

**Comment by Emaasit** [comment]
I absolutely understand. I'll handle most of the work load. Thanks.

**Comment by thomasp85** [comment]
Should I just merge my own PR, or how are we doing this?

**Comment by Emaasit** [comment]
I have merged it. Thanks.

File: ggraph.Rmd [hunk 1]
```diff
@@ -5,27 +5,159 @@ title: "ggplot2 extensions: ggraph"
...
-# All igraph layouts are available - here we use Fructerman and Reingold
-ggraph(graph = gr, layout = 'fr') + 
-    geom_edge_link(aes(size = weight), color = 'grey', alpha = 0.5) + 
-    geom_node_point(aes(color = class), size = 10) + 
-    coord_fixed() + 
-    ggforce::theme_no_axes()
+
+# To show a simple example of the API we consider a hierarchy
+## Create a graph of the flare class hierarchy
+flareGraph <- graph_from_data_frame(flare$edges, vertices = flare$vertices)
```
Comment (Emaasit) [inline ggraph.Rmd:30]: @thomasp85 I am getting an error here: "object 'flare' not found"

Comment (thomasp85) [inline ggraph.Rmd:30]: Have you updated ggraph?

File: ggraph.Rmd [hunk 2]
```diff
@@ -5,27 +5,159 @@ title: "ggplot2 extensions: ggraph"
...
+ggraph(flareGraph, 'igraph', algorithm = 'tree') +
```
Comment (Emaasit) [inline ggraph.Rmd:31]: I have just updated it and now I get this error:
"Error: ggplot2 doesn't know how to deal with data of class function"

Comment (thomasp85) [inline ggraph.Rmd:31]: Until ggplot2 2.1 hits cran you need to install from GitHub.

Comment (Emaasit) [inline ggraph.Rmd:31]: Thanks. I updated ggplot2 to version 2.1 and it run smoothly.

**Pull request merged**